bridge_adaptivity/module/templates/module/modals/delete_collection_alarm.html

```diff
@@ -0,0 +1,30 @@
+{% load bootstrap3 %}
+
+<!-- Modal -->
+<div class="modal fade" id="deleteModal{{ id }}" role="dialog">
+  <div class="modal-dialog" role="dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+          aria-hidden="true">&times;</span></button>
+          <h3 class="modal-title">This item is used by the student!</h3>
+      </div>
+      <div class="modal-body">
+          <h4>At least one student is still working on the collection: {{ object }}.</h4>
```

Comment:
I would add word `may`: "At least one student is still may working on the collection:"